### PR TITLE
#15 Responsive homepage

### LINKED ISSUE #15: Make the home page responsive

### PR DESCRIPTION
**Pull request by JagrajAulakh**
@Windsor-Code-Camp/frontend-devs Fix for #15 
The homepage is now responsive 😎 
check it out

`git checkout origin/#15-responsive`
`yarn start`

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
-				"secondary-color-800": "#A84C74",
-				"secondary-color-700": "#B16A86",
-				"secondary-color-600": "#B98095",
-				"secondary-color-500": "#C192A3",
-				"secondary-color-400": "#C8A1AF",
-				"secondary-color-300": "#CFAFBA",
-				"secondary-color-200": "#D6BCC4",
-				"secondary-color-100": "#DDC7CE",
+				"secondary-color": {
+					900: "#9F005D",
```
Comment (hassanuahmad) [inline tailwind.config.js]: this was supposed to be #9E005D I believe

File: src/components/HomePage/WhyWCC.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ function WhyWCC() {
 			</p>
 			<div>
 				{/* Point 1 */}
-				<div className="flex justify-center items-center py-32">
```
Comment (hassanuahmad) [inline src/components/HomePage/WhyWCC.js:15]: we will just create a array for it and map through it cuz that way we can just keep one div

**Comment by hassanuahmad** [comment]
Good work! 👍

File: src/components/HomePage/WhyWCC.js (hunk 2 above)
Comment (JagrajAulakh) [reply src/components/HomePage/WhyWCC.js:15]: Will get to it after this gets merged. I'm not trying to rebase the array change with this responsive stuff 😢

Comment (hassanuahmad) [reply src/components/HomePage/WhyWCC.js:15]: ok if u can fix the color, should be good to go then

File: tailwind.config.js (hunk 1 above)
Comment (JagrajAulakh) [reply tailwind.config.js]: Done

**Pull request merged**